game/scripts/vscripts/util.ts

```diff
@@ -347,7 +347,7 @@ export const createParticleAttachedToUnit = (particleName: string, unit: CDOTA_B
 
     return particleID;
 }
-export type HighlightType = "circle" | "arrow" | "arrow_enemy"
+export type HighlightType = "circle" | "arrow" | "arrow_enemy" | "dialogCircle"
```

Comment:
"dialog_circle" for same convention as the other one

Reply:
Changed to dialog_circle